soc: cc13x2_cc26x2: support setting CCFG values via Kconfig (#16147)
zephyrproject-rtos/zephyr: boards/arm/cc1352r1_launchxl/doc/index.rst

   :maybe-skip-config:
   :goals: debug

Bootloader
==========

> The ROM bootloader on the CC13x2 / CC26x2 devices is enabled by default. The

Reviewer: ```suggestion
The ROM bootloader on CC13x2 and CC26x2 devices is enabled by default. The
```

bootloader will start if there is no valid application image in flash or the

> so-called backdoor is enabled and BTN-1 is held down during reset. See the

Reviewer: ```suggestion
so-called backdoor is enabled (via option
:option:`CONFIG_CC13X2_CC26X2_BOOTLOADER_BACKDOOR_ENABLE`)
and BTN-1 is held down during reset. See the
```

> bootloader documentation in section ten of the `TI CC13x2 / CC26x2 Technical

Reviewer: ```suggestion
bootloader documentation in chapter 10 of the `TI CC13x2 / CC26x2 Technical
```

> CC2538 / CC26xx Serial Bootloader Interface`_ documentation for additional

Author: You're right. I will make these changes.
